.github/workflows/ci.yml

```diff
@@ -48,21 +36,7 @@ jobs:
           cache-dependency-path: '**/pnpm-lock.yaml'
       - run: pnpm install --frozen-lockfile
       - run: pnpm run compile
-      - name: Get installed Playwright version
-        id: playwright-version
-        run: |
-          PLAYWRIGHT_VERSION=$(pnpm playwright -V)
```

Comment:
windows don't support such syntax. And I realized that cache never hit for some reason, I decided to remove this for now. We can add it again when e2e spend more time (like 15 minutes in my previous project)